votor/src/consensus_pool_service/stats.rs

```diff
@@ -0,0 +1,96 @@
+use {
+    solana_metrics::datapoint_info,
+    std::time::{Duration, Instant},
+};
+
+const STATS_REPORT_INTERVAL: Duration = Duration::from_secs(10);
+
+#[derive(Debug)]
+pub(crate) struct ConsensusPoolServiceStats {
+    pub(crate) add_message_failed: u32,
+    pub(crate) certificates_sent: u16,
+    pub(crate) certificates_dropped: u16,
```

Comment:
is there a particular reason we're trying to get cute with the bytes here as opposed to just using `usize`? We end up casting all of these to 64 bit anyways when we report. I'm not sure the couple dozen bytes of savings are worth the complexity and multiple increment methods and such